src/validation/validator.ts

```diff
@@ -1 +1,23 @@
-export {}
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import yup from 'yup'
+
+export const validate = async ({
```

Comment:
change the export name to `validator`

Reply:
Thank you, very great review I will do it as soon as possible